java2typescript-jackson/src/test/resources/java2typescript/jackson/module/WriterPreferencesTest.enumToStringLiteralType.d.ts

```diff
@@ -3,5 +3,11 @@ export type Enum =
     | "VAL2"
     | "name";
 
+export interface TestClassHasMapWithEnumKey {
+    enumKeyMap: { [key in Enum ]: string;};
+    stringKeyMap: { [key: string ]: string;};
+    intKeyMap: { [key: string ]: string;};
```

Comment:
hmm, this looks like another bug I haven't noticed so far - should be `intKeyMap: { [key: number ]: string;};` instead. I created separate [issue](https://github.com/raphaeljolivet/java2typescript/issues/80) for that, so please remove `intKeyMap` and `stringKeyMap` fields from the test